extensions/wikia/PageHeader/PageTitle.class.php

```diff
@@ -40,14 +41,21 @@ public function __construct( WikiaApp $app ) {
 	private function handleTitle( WikiaApp $app ): string {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 7, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)